src/languages/es.js

```diff
@@ -729,6 +729,7 @@ export default {
             testTransactions: 'Transacciones de prueba',
             issueAndManageCards: 'Emitir y gestionar tarjetas',
             reconcileCards: 'Reconciliar tarjetas',
+            growlMessageOnCreate: 'Espacio de trabajo creado',
```

Comment:
@thesahindia I reckon this is from google translate since I can't find you asking about this on slack.

I've asked for the translation on [this 🧵](https://expensify.slack.com/archives/C01GTK53T8Q/p1646358687491589)

Reply:
Asked in our internal `#espanol` room and got the following translation: 

```
El espacio de trabajo ha sido creado
```

Reply:
Thanks! 🙇‍♂️
@thesahindia let's get this done :)